docs/beta/grain.md

```diff
@@ -6,17 +6,34 @@ image: img/uploads/grain-token-icon.png
 ## What is Grain?
 
 Grain is a community-specific digital currency that is issued on the basis of
-[Cred scores](https://sourcecred.io/docs/beta/cred). Grain represents how much a
-participant has supported that community, either through contributions or
-financial support.
+[Cred scores](https://sourcecred.io/docs/beta/cred).  If cred answers the question
+"*who provided value*?", Grain answers ***"how should we reward people given the value
+they provided?"***
+
+### What is the value of Grain?
+Every community will use Grain differently.  In our community, Grain is currently
+redeemable for Dai, a cryptocurrency that is worth 1 US Dollar.  Communities without
+funds might treat this as a share of future income.  Another might treat it as weight
+behind a community member's vote.
```

Comment:
Perhaps a sentence to the effect that the possibilities are really endless, and these suggestions aren't meant to narrow the use cases for Grain?

E.g.
"The only limit is your creativity."

Reply:
Love it